### DISCUSSION TITLE
FNIRS Signal

### DISCUSSION THREAD

**Discussion by shatton412** [compneuro-da/rsHRF#60] [General]
Can this application be used for FNIRS data?
Thank you, 
Shelby

**Comment by danielemarinazzo** [comment]
As long as you can make a reasonable assumption that some neural events elicit an hemodynamic response. And please make sure that the parameters (sampling rate, basis functions, etc.) are ok. Here https://www.frontiersin.org/articles/10.3389/fneur.2022.1038700/full they applied it to fNIRS.
